models/single.php

```diff
@@ -158,7 +158,7 @@ public function related() : ?array {
         );
 
         return [
-            'title' => get_field( 'related_title' ) ?? '',
+            'title' => get_field( 'related_title' ) ?? __( 'Related posts', 'tms-theme-base' ),
```

Comment:
Tässä kannattaa muistaa, että jos `get_field` palauttaa esim. `false` tai tyhjän stringin, niin tuo fallback ei tule voimaan. Se asetetaan vain, jos `get_field` palauttaa `null`in.